data-prepper-plugins/s3-sink/README.md

```diff
@@ -0,0 +1,9 @@
+Attached simple-sample-pipeline: opensearch-data-prepper\data-prepper\data-prepper-plugins\s3-sink\src\main\resources\pipelines.yaml

```

Comment:
Please look at other README.md files for the format of this file. And modify this file to be similar to those files.

Reply:
Thanks, @kkondaka for your review. I have modified README.md file as per other existing file format.